cmd/podman/exec.go

```diff
@@ -82,11 +85,34 @@ func execCmd(c *cliconfig.ExecValues) error {
 		return errors.Wrapf(err, "unable to exec into %s", args[0])
 	}
 
+	if c.PreserveFDs > 0 {
+		entries, err := ioutil.ReadDir("/proc/self/fd")
```

Comment:
Can you use `os.NewFile` here, and directly pass the FD number? The docs say it will ensure a file wrapping a valid FD is returned, so we can use it to check if the FD in question is present

Reply:
I think it means it checks only that once casted it is not <0:

```
    fmt.Printf("got: %v\n", os.NewFile(1, "f1"))
    fmt.Printf("got: %v\n", os.NewFile(100000000000, "f2"))
    fmt.Printf("got: %v\n", os.NewFile(9223372036854775808, "f3"))
```

I get:

```
got: &{0xc00006c180}
got: &{0xc00006c1e0}
got: <nil>
```

Reply:
Ah, so it's fake checking and the stdlib docs are lying, then.

It would have been neater than checking /proc, but if we can't do it this works fine.